.github/PULL_REQUEST_TEMPLATE/feature.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature (PR)
+about: template for a PR that implements a new feature 
+title: "Feature: [FEATURE NAME]"
+labels: 
+assignees:
+---
+
+Technology (Ansible, Helm, Terraform): TODO
```

Comment:
It's quite obvious by looking at the file changes which technology the PR touches on. What information do we gain here by requiring the submitter of a PR to fill this in again?

Reply:
Usually, one, first, tries to get some context before looking at the actual changes, to better understand them. At least, this is how would deal with new PRs. The templates are meant to make our life easier, so why not asking for this, too.